package.json

```diff
@@ -225,6 +225,10 @@
                 {
                     "command": "vscode-docker.registries.azure.runFileAsTask",
                     "when": "isWorkspaceTrusted"
+                },
+                {
+                    "command": "vscode-docker.help.openWalkthrough",
+                    "when": "never"
```

Comment:
There's already a VSCode command to open the walkthrough so I don't see a need for this to show in the palette.

Reply:
The only reason we need this command defined at all is because the tree view command can't provide an argument to `workbench.action.openWalkthrough`.